editor/plugins/node_3d_editor_plugin.h

```diff
@@ -186,8 +186,13 @@ class Node3DEditorViewport : public Control {
 	ViewType view_type;
 	void _menu_option(int p_option);
 	void _set_auto_orthogonal();
+
 	Node3D *preview_node;
 	AABB *preview_bounds;
+	Ref<Material> *preview_material;
+	Ref<Material> *preview_reset_material;
+	ObjectID *preview_material_target;
```

Comment:
Can use ObjectID directly.

Reply:
And the pointers to `Ref<T>` also seem like uncommon usage. `Ref<T>` is a smart pointer already.

Reply:
This should be resolved now